pydatalab/pydatalab/models/equipment.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+
+from pydantic import Field
+
+from pydatalab.models.items import Item
+
+
+class Equipment(Item):
+    """A model for representing an experimental sample."""
+
+    type: str = Field("equipment", const="equipment", pattern="^equipment$")
+
+    serial_numbers: Optional[str]
+    """A string describing one or more serial numbers for the instrument."""
```

Comment:
If it really is one or more

```suggestion
    serial_numbers: Optional[list[str]]
    """A string describing one or more serial numbers for the instrument."""
```

Reply:
I had also thought about this as well - the initial reasons for this being a string are similar to the reasons  previously discussed with hazards: its not trivial to make an ergonomic frontend input field for a list, and we just need a field to start keeping track of this stuff now.

But also, in this case, this is a field I envision will be mostly used by humans and could take different forms depending on what's useful (like, just: "XX123" or "XX123 (spectrometer), 789A/B (magnet)"). Making it a list could restrict the flexibilty here, and I don't think it really gives any benefits I can forsee (unlike hazard codes, where we can do cool things like display the pictographs once we properly keep track of them individually).

Reply:
This is probably fair enough, as long as we can make our free text search work with whatever format people provide (e.g., if its comma delimited or otherwise)